sdk/js/src/index.ts

```diff
@@ -27,12 +27,16 @@ export const initialize = (environmentKey: string, user: DVCUser, options: DVCOp
         throw new Error('Missing environment key! Call initialize with a valid environment key')
     }
 
+    if (!options || options === null) {
```

Comment:
should we check the user too

Reply:
yeah, will add another check and throw and error if the user doesn't meet the minimum required by the `DVCUser` type